articles/azure/azs-how-create-spn-portal.md

```diff
@@ -0,0 +1,112 @@
+---
+title: How to create a service principal name for Azure Stack using the Azure portal
+description: Learn how to create a service principal name (SPN) to manage your Azure Stack resources using the Azure portal
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a Service Principal Name
+toc_sub3:
+toc_sub4:
+toc_title: Create a service principal name - Portal
+toc_fullpath: Users/How To/Create a Service Principal Name/azs-how-create-spn-portal.md
+toc_mdlink: azs-how-create-spn-portal.md
+---
+
+# How to create a service principal name for Azure Stack using the Azure portal
+
+This document explains how to create a service principal name to manage Azure and Azure Stack using the Azure portal.
+
+It will guide you through the creation of:
+
+- An Azure application
+
+- A service principal name
+
+- Role assignment
+
+- Permissions
+
+## What is a service principal name
+
+An Azure service principal name (SPN) is a security identity used by user-created applications, services, and automation tools to access specific Azure resources. Think of it as a 'user identity' (username and password or certificate) with a specific role, and tightly controlled permissions. It only needs to be able to do specific things, unlike a general user identity. It improves security if you grant it only the minimum permissions level needed to perform its management tasks.
+
+To log in and manage your resources via SPN you'll need to create an Azure application and then assign an SPN to it. Only then will you be able to perform tasks against your environment.
+
+## Prerequisites
+
+- An active Azure Subscription
+
+## Create an SPN account for Azure and Azure Stack
```

Comment:
You don't need this extra heading. Remove it and promote the other sections to level 2 (##)